start-deployment-manager.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")"
+
+if [ "$#" -ne 2 ]; then
+    echo "Wrong number of arguments. Usage: <Assembly postfix> <deployment manager project directory>"
```

Comment:
`<deployment manager project directory>` is then called `LAUNCH_CONFIGURATION` later on. How come?
Also how come it's called `<Assembly postfix>` and not just assembly name?